### DISCUSSION TITLE
HATA : Bu sayfa mevcut değildir

### DISCUSSION THREAD

**Discussion by lodoskent** [Yaro2709/New-Star#21] [Q&A] [answered]
Merhaba Yaro 

Ben Filo alanına girdiğim de resimde olan yere bastığımda bu hatayı alıyorum. Yardımcı ol. 

![capture-20210115-234959](https://user-images.githubusercontent.com/39202729/104777375-9adaca80-578c-11eb-931f-2e2ef90d0512.png)
![capture-20210115-235023](https://user-images.githubusercontent.com/39202729/104777377-9b736100-578c-11eb-8c78-fc3b8a1242a1.png)

**Comment by Yaro2709** [comment, marked as answer]
Если страницы недоступны, то проверьте имена ссылок: например admin.php?page=configuni ИЛИ admin.php?page=Configuni ИЛИ admin.php?page=configUni ИЛИ admin.php?page=СonfigUni

**Reply by lodoskent** [reply]
Dediklerin nerede bulamadım. Ama çoğu yere baktım her şey normal gibi gözüküyor.

İstediğin sayfa olursa sana kodları gönderebilirim. Bakıp hatayı görmen için

**Reply by Yaro2709** [reply]
Этот недочет уже сдавали.

**Reply by lodoskent** [reply]
> Этот недочет уже сдавали.

Demek istediğini anlamadım. ama o kodları nasıl düzenleyebileceğimi söylersen yapabilirim. yada sende yapılı hali varsa bana gönder

**Reply by Yaro2709** [reply]
Скачать Notepad++ и открывать файлы и редактировать.

**Comment by bin9yuz79** [comment]
Merhaba, New Star 2.8.1 Türkçe dil dosyasını nasıl bulabilirim? @Yaro2709  @lodoskent
